airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4823,6 +4844,26 @@ components:
       properties:
         succeeded:
           type: boolean
+    AttemptNormalizationStatuses:
```

Comment:
nit: I think the convention in this file would be to name this `AttemptNormalizationStatusReadList` (see `WorkspaceReadList` as an example of the convention in this file)